I/IGLWrap/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+#=
+BINARYBUILDER_RUNNER=privileged julia build_tarballs.jl --verbose --debug       
+=#
+
+using BinaryBuilder
+using Pkg
+
+version = v"2.3.0" # libIGL version, see below:
+
+sources = [
+	DirectorySource("./iglwrap"),
+	# 2.3.0 stable release:
+	GitSource("https://github.com/libigl/libigl.git",
+		"e60423e28c86b6aa2a3f6eb0112e8fd881f96777"),
+	FileSource("https://www.gnu.org/licenses/gpl-3.0.txt",
+		"3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p build
+cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_MODULE_PATH=${WORKSPACE}/srcdir/libigl/cmake\; \
+    -DCGAL_test_cpp_version_RUN_RES=0 \
+    -DCGAL_test_cpp_version_RUN_RES__TRYRUN_OUTPUT="$(c++ -xc++ - -E -dM < /dev/null | grep -oP '__cplusplus \K.*')" \
+    ..
+make -j${nproc}
+make install
+install_license gpl-3.0.txt
```

Comment:
We actually already ship in the RootFS the GPL3 license:
```suggestion
install_license /usr/share/licenses/GPL3
```
and you don't need to add a `FileSource`